cypress/integration/ContributionForm_spec.js

```diff
@@ -0,0 +1,49 @@
+describe('ContributionForm', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
```

Comment:
lets get set up for stubbed data as soon as our be is ready

Reply:
We will get this functional ASAP <3